.github/workflows/release.yaml

```
@@ -0,0 +1,112 @@
+name: Promote Operator Release
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: 'Name for the tag of the release.'
+        required: true
+      quay_registry:
+        description: 'Quay registry to push to.'
+        default: 'quay.io/ansible'
+
+jobs:
+  promote-staged-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set TAG_NAME for workflow_dispatch event
+        if: ${{ github.event_name == 'workflow_dispatch' }}
+        run: |
+          echo "TAG_NAME=${{ github.event.inputs.tag_name }}" >> $GITHUB_ENV
+          echo "QUAY_REGISTRY=${{ github.event.inputs.quay_registry }}" >> $GITHUB_ENV
+
+      - name: Set TAG_NAME for release event
+        if: ${{ github.event_name == 'release' }}
+        run: |
+          echo "TAG_NAME=${{ github.event.release.tag_name }}" >> $GITHUB_ENV
+
+      - uses: actions/checkout@v3
+        with:
+          ref: 'main'
+          depth: 0
+          path: eda-server-operator
+
+      - name: Log into registry ghcr.io
+        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d    # v3.0.0
```

Just curious about why we're using the sha1 rather than the version (either short or full one)
```suggestion
        uses: docker/login-action@v3
```

---

explained here https://github.com/ansible/eda-server-operator/pull/161#discussion_r1445670410